crowbar_framework/app/helpers/barclamp/neutron_helper.rb

```diff
@@ -17,35 +17,15 @@
 
 module Barclamp
   module NeutronHelper
+
     def networking_plugins_for_neutron(selected)
       options_for_select(
-        [
-          ["ml2", "ml2"],
-          ["vmware", "vmware"]
-        ],
+        NeutronService.networking_plugins_valid.map{|el| [el, el]},
         selected.to_s
       )
     end
 
-    def networking_ml2_mechanism_drivers_for_neutron(selected)
-      selected = selected.gsub(/\s+/, "").split(",")
-      options_for_select(
-        [
-          ["linuxbridge", "linuxbridge"],
-          ["openvswitch", "openvswitch"],
-          ["cisco_nexus", "cisco_nexus"],
-        ],
-        selected
-      )
-    end
-
-    def networking_ml2_type_drivers_valid()
-      ["vlan", "gre"]
-    end
-
-    def networking_ml2_type_drivers_for_neutron(selected)
-      selected = selected.gsub(/\s+/, "").split(",")
-      valid_options = networking_ml2_type_drivers_valid()
+    def self.options_keep_order(valid_options, selected)
```

Comment:
Done in https://github.com/crowbar/barclamp-neutron/commit/ed72a9a35db400e62101fa774ed32947ee5880b1